### PR TITLE
add minimum distance method to base class and polygon env

### PR DESCRIPTION
**Pull request by veds12**

### CODE REVIEW

**Comment by threewisemonkeys-as** [comment]
Will merge after you change the `Point` to `shapely.geometry.Point` in polygon env as well the the lint check

File: gennav/envs/polygon_env.py [hunk 1]
```diff
@@ -94,7 +94,28 @@ def nearest_obstacle_distance(self, state, return_object=True):
...
+        """Function to get the minimum distance of each obstacle form the robot state
+
+        Uses shapely Point's distance method to obtain the minimum distances
+
+        Args:
+            state (gennav.utils.common.RobotState) : present state of the robot
+            sort (bool default = False) : returns the distances in ascending form when set true
+        Returns :
+            min_dist (list) : list containing minimum distances(float) of each obstacle from the robot state
+
```
Comment (threewisemonkeys-as) [inline gennav/envs/polygon_env.py]: Extra line in docstring here

**Comment by threewisemonkeys-as** [comment]
Are you sure you have black>=5.0 ?

**Comment by threewisemonkeys-as** [comment]
Also could you fix the lgtm alert?

**Comment by veds12** [comment]
Yeah I ll sort it out

**Comment by threewisemonkeys-as** [comment]
Is `tox -e lint .` working properly locally for you?

**Comment by veds12** [comment]
lint command fails when ran in the root directory

**Comment by veds12** [comment]
I already fixed the lgtm alert in the previous commit

**Pull request closed without merging**